homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -260,13 +298,15 @@ def device_class(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes of the sensor."""
+        print(self.current_observations)
+        print(self.observations_by_entity)
```

Comment:
Here are two print statements.

Reply:
nevermind, just saw removed 5 hours ago in https://github.com/home-assistant/core/pull/33916